src/opentime/timeRange.h

```diff
@@ -10,43 +10,46 @@
 
 namespace opentime { namespace OPENTIME_VERSION {
 
-/**
- * It is possible to construct TimeRange object with a negative duration.
- * However, the logical predicates are written as if duration is positive,
- * and have undefined behavior for negative durations.
- *
- * The duration on a TimeRange indicates a time range that is inclusive of the start time,
- * and exclusive of the end time. All of the predicates are computed accordingly.
- */
-
-/**
- * This default epsilon_s value is used in comparison between floating numbers.
- * It is computed to be twice 192khz, the fastest commonly used audio rate.
- * It can be changed in the future if necessary due to higher sampling rates
- * or some other kind of numeric tolerance detected in the library.
- */
+/// @brief This default epsilon_s value is used in comparison between floating numbers.
+///
+/// It is computed to be twice 192khz, the fastest commonly used audio rate. It
+/// can be changed in the future if necessary due to higher sampling rates or
+/// some other kind of numeric tolerance detected in the library.
```

Comment:
Might be worth noting that twice 192khz implies 1/2 a 192khz frame

Reply:
Sounds good, I updated the comment.